### DISCUSSION TITLE
Code formatting issues

### DISCUSSION THREAD

**Discussion by kryptus36** [zed-industries/zed#20115] [Feature Requests]
I'm struggling a bit to get the code formatter to not do silly things.

![image](https://github.com/user-attachments/assets/c33c6b91-019e-4fc9-a177-4f80d5b2fede)

Anybody with suggestions on where I could configure it to not put in line breaks before a closing angle brace?

**Comment by xnousnow** [comment]
Try the methods in prettier/prettier#6460 :)

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
